src/ContainerDetails.jsx

```diff
@@ -56,10 +56,6 @@ const ContainerDetails = ({ container, containerDetail }) => (
             <DescriptionListTerm>{_("IP address")}</DescriptionListTerm>
             <DescriptionListDescription>{containerDetail ? containerDetail.NetworkSettings.IPAddress : ""}</DescriptionListDescription>
         </DescriptionListGroup>
-        <DescriptionListGroup>
```

Comment:
I am not fan of this commit. If we have that info, why not show it? We also have space for it, it does not cost us anything...

Reply:
Oh, should specify that I mean the very first one dropping prefix length

Reply:
It will just be 0 most of the cases and it will always shown. How someone should use that anyway?

Reply:
:man_shrugging: ok, then we can drop it